src/Router.js

```diff
@@ -0,0 +1,18 @@
+import { BrowserRouter,Route,Routes } from "react-router-dom";
+import {Login,Profile} from "./page/index"
+const Router = () =>{
+   return(
+       <>
+        <BrowserRouter>
+    <Routes>
+        <Route path="" element={<Login/>}  />
+    </Routes>
```

Comment:
You just have to make one parent Routes of all route

Reply:
Yes i have made those changed in recent commit